### DISCUSSION TITLE
Glances widgets inop after updating to Glances V4 which now uses fastAPI

### DISCUSSION THREAD

**Discussion by marcmcmillin** [Lissy93/dashy#1600] [Help and Support] [answered]
I recently upgraded Glances on my Ubuntu 22.04 LTS Proxmox PVE VMs. V4 now uses fastAPI for the webserver and the widgets for Glances no longer work.

Thanks in advance for any assistance

Mark

![image](https://github.com/Lissy93/dashy/assets/59179779/ee98f511-e65c-4f47-bf58-bdf01ea62f2e)

**Comment by React75** [comment, marked as answer]
Same issue.

There's some info here: https://dashy.to/docs/troubleshooting#widget-cors-errors

**Reply by marcmcmillin** [reply]
@React75 are you indicating that this is a CORS issue?

**Reply by marcmcmillin** [reply]
I installed the Allow CORS extension in Chrome and that fixed the issue. Thanks! @React75

**Comment by React75** [comment]
I found this worked:

> Just add the useProxy: true option to the failing widget.
